resources/jscomposition/cases/casesDetail/config/columns.js

```diff
@@ -78,10 +83,11 @@ const requestNumberColumn = () => ({
 });
 
 const requestNameColumn = () => ({
```

Comment:
This column does not exist
![image](https://github.com/user-attachments/assets/1664711a-887a-4617-8e7d-f16c78d3a73b)